getting_started/step_by_step/exporting.rst

```diff
@@ -101,10 +101,8 @@ changed:
             $AnimatedSprite.stop()
 
         position += velocity * delta
-        # We don't need to clamp the player's position
```

Comment:
While these comments are not relevant, something along the lines of your explanation regarding aspects and screen resizing would not go amiss.

Reply:
Since the code part I modified starts with the following line I thought that no comments were necessary.
> Here is the full script for the player, with comments noting what we've changed:


But maybe you are right, would something like this improve the code?
```
position += velocity * delta
# We still need to clamp the player's position here because Godot tries to 
# keep the aspect ratio by creating black borders if necessary.
# Without clamp the player would be able to move under those borders.
position.x = clamp(position.x, 0, screen_size.x)
position.y = clamp(position.y, 0, screen_size.y)
```

Or do you have something different on your mind?
(Please feel free to improve the text, I am not a native English speaker)

Reply:
I would slightly expand this to 

```
# We still need to clamp the player's position here because on devices that don't
# mach your game's aspect ratio Godot will try to maintain it as much as possible 
#  by creating black borders if necessary.
# Without clamp the player would be able to move under those borders.
```

it gives a little more information as to why this could be a case and why does Godot create black bars out of nowhere

Reply:
I added the comments to explain why the clamp is still needed.